test/ShapeCrawler.Tests.Unit/ShapeCollectionTests.cs

```diff
@@ -347,6 +348,69 @@ public void AddPicture_adds_svg_picture()
         pres.Validate();
     }
 
+    [Test]
+    public void AddPicture_svg_should_not_duplicate_the_image_source_When_the_same_image_is_added_twice()
+    {
+        // Arrange
+        var pres = new Presentation();
+        var shapes = pres.Slides[0].Shapes;
+        var image = StreamOf("test-vector-image-1.svg");
+
+        // Act
+        shapes.AddPicture(image);
+        shapes.AddPicture(image);
+
+        // Assert
+        var checkXml = SaveAndOpenPresentationAsXml(pres);
+        var imageParts = checkXml.PresentationPart.SlideParts.SelectMany(x=>x.ImageParts).ToArray();
+        imageParts.Length.Should().Be(2);
+        SavePresentationFile(pres);
```

Comment:
As mentioned [here](https://github.com/ShapeCrawler/ShapeCrawler/issues/803#issuecomment-2564385180), please avoid using Hard Drive if the test doesn't explicitly test integration with Hard Drive.